pyscal/gaswater.py

```diff
@@ -245,30 +245,30 @@ def crosspoint(self):
             float: the gas saturation where krw == krg, for relperm
                 linearly interpolated in water saturation.
         """
-        if not {"sw", "krw"}.issubset(self.wateroil.table.columns):
+        if not {"SW", "KRW"}.issubset(self.wateroil.table.columns):
             logger.warning("Can't compute crosspoint when krw is not present")
```

Comment:
Should the warning explicitly use CAPS LOCK for the missing fields to make it crystal clear; when krw is not ->when KRW is not? As of now the user might think: but I do have krw!!  Not realizing that capitalization of keys is the problem.

Reply:
Will fix. The PR has been made with `perl -p -i -e 's/"sw"/"SW"/g' *.py`..